tests/foreman/ui/test_contentview.py

```diff
@@ -644,641 +369,94 @@ def test_positive_update_name(self):
-                session, org=self.organization.name, name=name)
-            self.assertIsNotNone(self.content_views.search(name))
-            # invalid_names_list is used instead of invalid_values_list
-            # because save button will not be enabled if name is blank
-            for new_name in invalid_names_list():
-                with self.subTest(new_name):
-                    self.content_views.update(name, new_name)
-                    self.assertIsNotNone(self.content_views.wait_until_element(
-                        common_locators['alert.error_sub_form']))
-                    self.assertIsNone(self.content_views.search(new_name))
-
-    @run_only_on('sat')
-    @tier1
-    def test_positive_update_description(self):
-        """Update content views description to valid one.
-
-        :id: f5e46a3b-c317-4575-9c66-ef1da1926f66
-
-        :expectedresults: Content view is updated successfully and has proper
-            description
-
-        :CaseImportance: Critical
-        """
-        name = gen_string('alpha', 8)
-        desc = gen_string('alpha', 15)
-        with Session(self) as session:
-            make_contentview(
-                session,
-                org=self.organization.name,
-                name=name,
-                description=desc,
-            )
-            self.assertIsNotNone(self.content_views.search(name))
-            for new_desc in valid_data_list():
-                with self.subTest(new_desc):
-                    self.content_views.update(name, new_description=new_desc)
-                    self.content_views.search_and_click(name)
-                    self.content_views.click(
-                        tab_locators['contentviews.tab_details'])
-                    self.assertEqual(
-                        self.content_views.wait_until_element(
-                            locators['contentviews.fetch_description']).text,
-                        new_desc
-                    )
-
-    @run_only_on('sat')
-    @tier1
-    def test_positive_delete(self):
-        """Delete content views
-
-        :id: bcea6ef0-bc25-4cc7-9c0c-3591bb8810e5
-
-        :expectedresults: Content view can be deleted and no longer appears in
-            UI
-
-        :CaseImportance: Critical
-        """
-        with Session(self) as session:
-            for name in valid_data_list():
-                with self.subTest(name):
-                    make_contentview(
-                        session, org=self.organization.name, name=name)
-                    self.assertIsNotNone(
-                        self.content_views.search(name),
-                        'Failed to find content view %s from %s org' % (
-                            name, self.organization.name)
-                    )
-                    self.content_views.delete(name)
-
-    @run_in_one_thread
-    @run_only_on('sat')
-    @skip_if_not_set('fake_manifest')
-    @tier2
-    def test_positive_add_rh_custom_spin(self):
```

Comment:
That case has basically no sense neither in term of coverage (as 100% of it covered in `test_positive_edit_rh_custom_spin`) nor in term of logic